config/crossSections.yaml

```
@@ -0,0 +1,619 @@
+ST_tW_antitop:
+  crossSec: 35.85
+  reference: https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#Single_top
+  unc: 35.85 +0.90 -0.90 +1.70 -1.70
```

if you add uncertainties, use machine readable format and add name of the sources. If unknown, better to not put unc values at all to not create confusion. E.g. could be something like (numbers/names are not correct, just for illustration):
```suggestion
  unc:
    scale: 0.9
    pdf:
      up: 1.7
      down: 2.2
```

---

PS
- if it is only one uncertainty value I'm fine with having
  ```
  unc: 12.345
  ```
- remove empty `unc:`
- if uncertainty is specified as Y %, put
  ```
  unc: Y * XS
  ```